src/com/trollworks/gcs/character/GURPSCharacter.java

```diff
@@ -2722,17 +2722,19 @@ public int getIntegerBonusFor(String id, StringBuilder toolTip) {
      * @param id                      The feature ID to search for.
      * @param nameQualifier           The name qualifier.
      * @param specializationQualifier The specialization qualifier.
+     * @parem categoryQualifier The categories qualifier
      * @return The bonus.
      */
     public int getSkillComparedIntegerBonusFor(String id, String nameQualifier, String specializationQualifier, String categoryQualifier) {
-        return getSkillComparedIntegerBonusFor(id, nameQualifier, specializationQualifier, null);
+        return getSkillComparedIntegerBonusFor(id, nameQualifier, specializationQualifier, categoryQualifier, null);
     }
 
     /**
      * @param id                      The feature ID to search for.
      * @param nameQualifier           The name qualifier.
      * @param specializationQualifier The specialization qualifier.
-     * @param toolTip                 The toolTip being built
+     * @parem categoryQualifier The categories qualifier
```

Comment:
Same here

Reply:
Ok, I think I have spell checking enabled correctly... although that is no guarantee that I will spell things right.   ;-)

Reply:
The JavaDoc verifier should actually be throwing a fit on these... but I must admit I haven't specifically tested that theory out.